Dockerfile

```diff
@@ -32,7 +32,14 @@ RUN set -ex; \
   ; \
   \
   # install the PHP extensions we need
-  docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr --with-webp-dir=/usr; \
+  # GD configuration changed in PHP >7.4. See: https://github.com/docker-library/php/pull/910#issuecomment-559383597
+  # test if $PHP_VERSION > 7.4 (source: https://stackoverflow.com/a/24067243)
+  if [ "$(printf '%s\n' $PHP_VERSION 7.4 | sort -V | head -n 1)" != "$PHP_VERSION" ]; then \
```

Comment:
@yannlugrin do you know if there is better way to manage small changes between image version while keeping a `Dockerfile` reference ?